deployments/package.json

```diff
@@ -1,16 +1,12 @@
 {
   "name": "@nexusmutual/deployments",
-  "version": "2.1.1",
+  "version": "2.2.0",
```

Comment:
We keep this in sync with the main package json since the deployment package reflects the smart contracts version so bumping this independently is undersirable.

Reply:
Oh, I didn't know. I did publish it already. Let's do a bump soon and sync up